test/functional/testlib.bash

```diff
@@ -1942,6 +1961,8 @@ update_bundle() {
 		sudo bsdiff "$oldversion_path"/files/"$fhash" "$version_path"/files/"$new_fhash" "$version_path"/delta/"$delta_name"
 		# create or add to the delta-pack
 		add_to_pack "$bundle" "$version_path"/delta/"$delta_name" "$oldversion"
+		# keep the modified file for the iterative "from manifest"
+		from_manifest_content="$(awk "/....\t.*\t.*\t${fname////\\/}/" "$oldversion_path"/Manifest."$bundle")"$'\n'
```

Comment:
Not sure what the `from_manifest_content` value looks like here. Specifically, I don't understand how this form of pattern substitution work. The awk usage looks okay if you want to print contents (space separated) of the manifest body, omitting the manifest header.

Reply:
The _from_manifest_content_ will have only the line from the manifest that was modified during the bundle update. The awk is looking for lines in the manifest that match this pattern:
<any four characters><a tab character><any string><a tab character><another string><a tab character><the name of the file, e.g. /foo/bar>

I guess the part of that pattern that is confusing is this ${fname////\\/}, which as you pointed out is a bash pattern substitution which is going to replace every "/" character with "\/", so basically is just escaping the \

Reply:
I will replace the occurrence of that pattern substitution with another variable so it only shows once, and I will add a comment about what that substitution is doing so the code is easier to read.